vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -2,52 +2,60 @@
   --ag-background-color: var(--main-container-bg-color);
   --ag-foreground-color: var(--text-primary);
   --ag-odd-row-background-color: var(--main-container-bg-color);
+  --ag-header-foreground-color: var(
+    --text-secondary
+  ); /* Colour of text and icons in the header */
+
+  --ag-data-color: var(--text-primary); /* Colour of text in grid cells */
   --ag-header-background-color: var(--main-container-bg-color);
   --ag-header-column-resize-handle-display: none;
-  --ag-borders: none;
   --ag-icon-font-family: aggridquartz;
   --ag-icon-size: 14px;
-  --ag-row-height: 48px; /* This determines the row height, but not header */
+  --ag-row-height: 48px;
   --ag-header-height: 40px;
+
+  /* Borders */
+  --ag-borders: none;
+  --ag-row-border-style: solid;
+  --ag-row-border-color: var(--border-subtle-alpha-01);
+  --ag-row-border-width: 1px;
+
+  /* Bg Color */
+  --ag-selected-row-background-color: var(--state-overlays-selected);
+  --ag-active-color: var(--state-overlays-selected-hover);
 }
 
-/* General setting header row */
 #page-container .ag-theme-quartz .ag-header-row {
-  color: var(--text-secondary);
-  font-size: 14px;
-  font-weight: normal;
+  align-items: flex-start;
+  border-bottom: 1px solid var(--border-subtle-alpha-02);
+  display: flex;
+  font-weight: 400;
+  padding: var(--spacing-03) 0 0 0;
 }
 
-/* Border at the bottom and padding */
+/* Header */
 #page-container .ag-theme-quartz .ag-header-cell {
-  border-bottom: 1px solid var(--border-subtle-alpha-02);
-  padding-left: 0;
-  padding-right: 12px;
+  align-items: center;
+  display: flex;
+  padding: 0 var(--spacing-03);
 }
 
-/* Border at bottom when hovering more solid */
-#page-container .ag-theme-quartz .ag-header-cell:hover {
-  border-bottom: 1px solid var(--border-hover);
+#page-container .ag-header-cell-text {
+  letter-spacing: -0.112px;
+  line-height: var(--spacing-04);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
Not sure if you saw these specifications in figma, but basically you have to click directly on the text and then you get these specifications. They now charge for dev mode - so you might have to get the license?

![Screenshot 2024-02-22 at 18 06 13](https://github.com/mckinsey/vizro/assets/90609403/9b9a54b9-33db-4d46-b8b1-fdf189aaf9af)